types/dd-trace/dd-trace-tests.ts

```diff
@@ -1,35 +1,52 @@
-import * as tracer from 'dd-trace';
-import SpanContext = require('dd-trace/src/opentracing/span_context');
+import * as tracer from "dd-trace";
+import SpanContext = require("dd-trace/src/opentracing/span_context");
 
 tracer.init({
-    service: 'MyLovelyService',
-    hostname: 'localhost',
+    service: "MyLovelyService",
+    hostname: "localhost",
     port: 8126,
     logger: {
-        debug: msg => { },
-        error: err => { }
-    }
+        debug: msg => {},
+        error: err => {},
+    },
+});
+
+tracer.use("express", {
+    service: "incoming-request",
+    headers: ["User-Agent"],
+    validateStatus: code => code !== 418,
```

Comment:
@andy-ms Any idea why on `@next` the parameters default to `any` instead of getting [the function signature typing](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/29013/files#diff-82e36869a1fa26fd7d4cb76e1a48c92bR211) applied like with released TS versions?

https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/430495451#L3613

Reply:
Microsoft/TypeScript#27224